ath.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+cd $(dirname $0)
+
+# https://github.com/jenkinsci/acceptance-test-harness/releases
+ATH_VERSION=5458.v911b_2f0818ee
+
+# TODO use Artifactory proxy?
+
+[ -f war/target/jenkins.war ] || mvn -B -ntp -Pquick-build -am -pl war package
+
+mkdir -p target/ath-reports
+chmod a+rwx target/ath-reports
+
+docker run --rm \
+  -e ATH_VERSION=$ATH_VERSION \
```

Comment:
```suggestion
  --env ATH_VERSION \
```

- Prefer long flag when they are available (easier to understand at a glance)
- If only providing the variable key, then docker forward the value from the current shell context. Require exporting the variable (see other comment).

Reply:
https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file FTR